blocks/src/checkout/index.tsx

```diff
@@ -11,14 +11,15 @@ import { decodeEntities } from "@wordpress/html-entities";
 import { registerPaymentMethod } from "@woocommerce/blocks-registry";
 // @ts-ignore - Cant avoid this issue, but its loaded in by Webpack
 import { getSetting } from "@woocommerce/settings";
+import { applyFilters } from "@wordpress/hooks";
```

Comment:
You can remove this, since it wont do anything other then just add a dependency that we don't actually have right now.